dev-util/bats/bats-1.4.1.ebuild

```diff
@@ -13,15 +13,24 @@ SRC_URI="https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="test"
 
 DEPEND="app-shells/bash:*"
 RDEPEND="${DEPEND}"
 
 S="${WORKDIR}/${MY_PN}-${PV}"
 
+src_prepare() {
+	default
+	if use test ; then
+		# "parallelity factor is met exactly" fails in portage, disable
+		sed -i -e 's/@test "parallelity factor is met exactly" {$/@test "parallelity factor is met exactly" { skip "gentoo"/' test/parallel.bats
+	fi
+}
+
 src_test() {
 	local my_jobs=$(makeopts_jobs)
-	if ! command -v parallel; then
+	if ! command -v parallel >/dev/null; then
```

Comment:
Why not simply have sys-process/parallel as test DEPEND? (Then IUSE="test" would be required again)

Reply:
this commit is not related to the bug, its just a drive-by fix

parallel is an optdepend, and depending on whether or not it is installed testing needs to be done differently

Reply:
I get that testing needs to be done if it's not installed, but I wonder why not simply have `DEPEND=test? ( sys-process/parallel )`?